src/admin/content/content.types.ts

```diff
@@ -21,12 +21,17 @@ export type ContentParams = { id: string };
 export interface ContentEditFormState {
 	title: string;
 	description: string;
+	isProtected: boolean;
 	path: string;
 	contentType: string;
 	publishAt: string;
 	depublishAt: string;
 }
 
+export interface ContentEditFormErrors extends Partial<Omit<ContentEditFormState, 'isProtected'>> {
```

Comment:
dit is wss wat mooier:
```
export type ContentEditFormErrors = Partial<{ [prop in keyof ContentEditFormState]: string }>;
```

Reply:
Klopt, ik wou persé een interface behouden 😄 i will change it.